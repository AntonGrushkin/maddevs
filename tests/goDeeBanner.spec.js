import {
  mount
} from '@vue/test-utils';
import GoDeeBanner from '@/components/GoDee/Banner';

describe('GoDee banner', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(GoDeeBanner);
  });

  // ------ IMPORTANT ----- //
  test('is a Vue instance', () => {
    expect(wrapper.exists()).toBeTruthy();
  });

  test('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
  // --------------------- //
});
