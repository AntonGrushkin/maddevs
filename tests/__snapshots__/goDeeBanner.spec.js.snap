// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoDee banner renders correctly 1`] = `
<section
  class="banner"
  id="banner"
>
  <div
    class="container-godee"
  >
    <div
      class="banner__banner-content"
    >
      <div
        class="banner__text-content"
      >
        <h1
          class="banner__main-title"
        >
          <img
            alt="GoDee"
            class="banner__main-title-logo"
            src="@/assets/img/GoDee/svg/goodee-logo.svg"
          />
        </h1>
         
        <h3
          class="banner__sub-title"
        >
          Convenient shuttle 
          <br />
           bus service
        </h3>
         
        <p
          class="banner__description"
        >
          Mad Devs helped GoDee to develop feature-rich 
          <br />
           software to re-invent public mobility by building new 
          <br />
           smart ways of daily commute.
        </p>
      </div>
       
      <div
        class="banner__phone-wrapper"
      >
        <img
          alt="preview"
          class="banner__phone-image"
          src="@/assets/img/GoDee/svg/godee-iphonex.svg"
        />
      </div>
       
      <img
        alt="Route"
        class="banner__route-line"
        src="@/assets/img/GoDee/svg/banner-godee-route-line.svg"
      />
    </div>
  </div>
</section>
`;
